src/components/molecules/ScheduleEvent/ScheduleEvent.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { MouseEventHandler, useCallback } from "react";
+import classNames from "classnames";
+import { useCss } from "react-use";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faBookmark as solidBookmark } from "@fortawesome/free-solid-svg-icons";
+import { faBookmark as regularBookmark } from "@fortawesome/free-regular-svg-icons";
+
+import { SCHEDULE_HOUR_COLUMN_WIDTH_PX } from "settings";
+
+import { PersonalizedVenueEvent } from "types/venues";
+
+import { isEventLive } from "utils/event";
+
+import { ONE_HOUR_IN_MINUTES } from "utils/time";
+
+import {
+  addEventToPersonalizedSchedule,
+  removeEventFromPersonalizedSchedule,
+} from "api/profile";
+import { useUser } from "hooks/useUser";
+
+import { calcStartPosition } from "components/molecules/Schedule/Schedule.utils";
+
+import "./ScheduleEvent.scss";
+
+export interface ScheduleEventProps {
+  event: PersonalizedVenueEvent;
+  scheduleStartHour: number;
+  personalizedEvent?: boolean;
+}
+
+export const ScheduleEvent: React.FC<ScheduleEventProps> = ({
+  event,
+  scheduleStartHour,
+  personalizedEvent: isPersonalizedEvent = false,
+}) => {
+  const { userId } = useUser();
+
+  // @debt ONE_HOUR_IN_MINUTES is deprectated; refactor to use utils/time or date-fns functions
+  const eventWidth =
+    (event.duration_minutes * SCHEDULE_HOUR_COLUMN_WIDTH_PX) /
+    ONE_HOUR_IN_MINUTES;
+
+  const containerCssVars = useCss({
+    "--event--margin-left": `${calcStartPosition(
+      event.start_utc_seconds,
+      scheduleStartHour
+    )}px`,
+    "--event--width": `${eventWidth}px`,
+  });
```

Comment:
This `calcStartPosition` would be better in a `const` I think. Would make this easier to read.

Reply:
moved to `eventMarginLeftPx` const : https://github.com/sparkletown/sparkle/pull/1426/commits/880f966fdbeb06065f5946456902e13410b88582